src/core_plugins/kibana/server/tutorials/netflow/common_instructions.js

```diff
@@ -66,25 +62,23 @@ export const COMMON_NETFLOW_INSTRUCTIONS = {
     OSX: [
       {
         title: 'Run the Netflow module',
-        textPre: 'In the Logstash installation directory, run the following command to set up the Netflow module.',
+        textPre: 'From the installation directory, run:',
```

Comment:
We removed references to the "installation directory" earlier in this file. Should this one be removed too, for consistency?

Reply:
Yes, for consistency.